core/state_connector_test.go

```diff
@@ -0,0 +1,153 @@
+package core
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/flare-foundation/coreth/core/types"
+	"github.com/flare-foundation/coreth/core/vm"
```

Comment:
Why group ethereum and flare packages together?

Usually we group the following way:

* standard library
* external packages
* internal packages

And optionally if there is a strong close dependency (like for Flow packages for the Flow DPS, or here ethereum packages I assume) we have a group specifically for it between external and internal packages.

But maybe it's done this way here for consistency with other files in the repo?